### DISCUSSION TITLE
Run code after install

### DISCUSSION THREAD

**Discussion by Refzlund** [vercel/turborepo#3761] [Help] [answered]
Hey👋

I need to run some code which will run on multiple machines after an (pnpm) install happens.
It's important that messages are visible to distinguish issues that may occur.

An example here being database migration.

### Here's my expected output:
![image](https://user-images.githubusercontent.com/21252096/218291901-fda69313-bf42-41d5-8911-2ac9d59f80a0.png)

### However, applying this script at `postinstall` inside `package.json` results in this:
![image](https://user-images.githubusercontent.com/21252096/218291927-b0d42a65-48a2-4a2a-a8c1-70a203d540c2.png)

-----------

I hope I can get some guidance on what the standardization for TurboRepo is here. I might be doing things wrong, so please give me your best shot! Thank you in advance🙏

Note, that this `package.json` belongs to a package submodule called `jobs` inside the monolith.

**Comment by mehulkar** [comment, marked as answer]
Hey @Refzlund, turborepo isn't responsible for installing your dependencies (and thus also not responsible for running `postinstall` scripts). Could you say more about how you think this is related to Turborepo?
